packages/apps-config/src/api/spec/storagehub.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright 2017-2024 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { storageHubDefinitions } from '@storagehub/types-bundle';
+
+export default storageHubDefinitions;
```

Comment:
Why is this needed and where is `storageHubDefinitions` coming from?

Reply:
This is importing all the manual definitions from the storageHub repo into polkadotJS. this way we wont have to keep changing this repo, and we can just up the package ver when our rpcs change